.github/workflows/ci.yml

```
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  # Allow manual runs. It is necessary to run the workflow manually, when the "formatter" workflow applies any changes.
+  workflow_dispatch:
+
+jobs:
+  Processor_Partial:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            framework: net9.0
+          - os: ubuntu-latest
+            framework: net8.0
+          - os: windows-latest
+            framework: net9.0
+          - os: windows-latest
+            framework: net8.0
+          - os: windows-latest
+            framework: net462
+          - os: macos-latest
+            framework: net9.0
+          - os: macos-latest
+            framework: net8.0
+          - os: macos-13
+            framework: net9.0
+          - os: macos-13
+            framework: net8.0
```

This can actually be simplified like this. It will generate the expected matrix and then add net462 for windows only.

```suggestion
        os: [ubuntu-latest, windows-latest, macos-latest, macos-13] # macos-13 for x86_x64 arch
        framework: [net8.0, net9.0]
        include:
          - os: windows-latest
            framework: net462
```